ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -90,6 +90,11 @@ export class Send extends IronfishCommand {
       default: false,
       description: 'Allow offline transaction creation',
     }),
+    notes: Flags.string({
```

Comment:
If you are using the multiple flag, this should be `note`

Reply:
you can also `--notes 72e91d6672d79f9006f128a16cf52bbb585037fee2aea8385b56919c3ac9570b 4a5feb193d425fa73abfb108027a643eb027e46684c50e2bef5d66ea48ec3d2d f8b91f81caf4489c83734c2b29ad4587c9ba0c1080854a3f6da19f6fad49265e` which I think is slightly simpler

Reply:
I looked at the docs and I think it seems singular naming conventions are standard, so I'll change it to `--note`